pkg/node/node.go

```diff
@@ -273,14 +274,20 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 
 	lightNodes := lightnode.NewContainer()
 
-	p2ps, err := libp2p.New(p2pCtx, signer, networkID, swarmAddress, addr, addressbook, stateStore, lightNodes, logger, tracer, libp2p.Options{
+	txHash, err := getTxHash(stateStore, logger, o.Transaction)
+	if err != nil {
+		return nil, err
```

Comment:
there should be a better error here. if nothing was specified and there was no deployment tx either bee will terminate with a storage.NotFound which does not tell the user much.